### DISCUSSION TITLE
posts/how-to-pass-value-from-javascript-to-php/

### DISCUSSION THREAD

**Discussion by dcangulo** [dcangulo/davidangulo.xyz#134] [Announcements]
<h1>posts/how-to-pass-value-from-javascript-to-php/</h1><a href='https://www.davidangulo.xyz/posts/how-to-pass-value-from-javascript-to-php/'>https://www.davidangulo.xyz/posts/how-to-pass-value-from-javascript-to-php/</a>

**Comment by NaveenNandhu** [comment]
Sir I receive the error 

Warning: Undefined array key "myvar" in C:\xampp\htdocs\hospro\Example\example.php on line 3

What can Do Sir?
